lib/utils.js

```diff
@@ -30,3 +31,18 @@ exports.joinListConjuction = function (list, separator, conjuction) {
   }
   return joinedList;
 };
+
+exports.makeGetStationResponseText = function (trainArrivals, stationName) {
+  var responseText = '';
+  if (_.size(trainArrivals) === 0) {
+    responseText = props.stationNoArrivalsSpeechOutput;
+  } else if (_.size(trainArrivals) === 1) {
+    var destinationStationName = _.keys(trainArrivals)[0];
+    responseText = 'The next train to ' + destinationStationName + ' arrives in ' +
+      trainArrivals[destinationStationName] + ' minutes. There are no other trains arriving at ' + stationName +
+      ' in the next 20 minutes.';
+  } else {
+    responseText = 'Are you going to ' + this.joinListConjuction(_.keys(trainArrivals), ', ', ' or ');
```

Comment:
What does the trainArrivals object look like? Could you have two `Vienna` keys? If so, having a `size > 1` does not always mean you need to ask the user the direction.